test/plz_run_env/env_printer.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+printenv DATA
```

Comment:
It's part of coreutils. I kinda like it because it seems more descriptive e.g. only works on env variables (as opposed to the various other bash variables): 

https://man7.org/linux/man-pages/man1/printenv.1.html

Guess it doesn't make much difference.